etl/steps/data/garden/eth/2023-03-15/ethnic_power_relations.meta.yml

```diff
@@ -0,0 +1,413 @@
+        display:
+          name: Number of marginalized ethnic groups (MEG)
+          numDecimalPlaces: 0
+        sources:
+          - *source-epr
+      regaut_groups_count:
+        title: Number of groups with regional autonomy
+        description: Count variable indicating number of groups with regional autonomy in this country.
+        unit: ""
+        short_unit: ""
+        display:
+          name: Number of groups with regional autonomy
+          numDecimalPlaces: 0
+        sources:
+          - *source-epr
+      regaut_excl_groups_count:
+        title: Number of marginalized ethnic groups (MEG) with regional autonomy
+        description: Count variable indicating number of MEG groups with regional autonomy in this country.
+        unit: ""
+        short_unit: ""
+        display:
+          name: Number of MEG groups with regional autonomy
+          numDecimalPlaces: 0
+        sources:
+          - *source-epr
+      regaut_egip_groups_count:
+        title: Number of ethnic groups in power (EGIP) with regional autonomy
+        description: Count variable indicating number of EGIP groups with regional autonomy in this country.
+        unit: ""
+        short_unit: ""
+        display:
+          name: Number of EGIP groups with regional autonomy
+          numDecimalPlaces: 0
+        sources:
+          - *source-epr
+      rlvt_groups_count:
+        title: Number of relevant groups
+        description: Count variable indicating the number of relevant groups in this country.
+        unit: ""
+        short_unit: ""
+        display:
+          name: Number of relevant groups
+          numDecimalPlaces: 0
+        sources:
+          - *source-epr
+      actv_groups_count:
+        title: Number of active groups
+        description: Count variable indicating the number of active groups in this country.
+        unit: ""
+        short_unit: ""
+        display:
+          name: Number of active groups
+          numDecimalPlaces: 0
+        sources:
+          - *source-epr
+      lpop:
+        title: Share of population ethnically relevant
+        description: Sum of the ethnically relevant population in this country (as a fraction of total population).
+        unit: "%"
+        short_unit: "%"
+        display:
+          name: Share of population ethnically relevant
+          numDecimalPlaces: 1
+
+      egippop:
+        title: Share of population in ethnic groups in power (EGIP)
+        description: Sum of the population of all EGIP groups in this country (as a fraction of total population).
+        unit: "%"
+        short_unit: "%"
+        display:
+          name: Share of population in EGIP groups
+          numDecimalPlaces: 1
+        sources:
+          - *source-epr
+      legippop:
+        title: Share of ethnically relevant population in ethnic groups in power (EGIP)
+        description: EGIP population as a fraction of ethnically relevant population in this country (egippop / lpop).
```

Comment:
Refer to variables by their title, not their name.

Reply:
That's right, thanks!